docker-compose.prod.yml

```diff
@@ -0,0 +1,97 @@
+# NOTE: This files shows a possible production setup.
+#       If you are not familiar with handling containers in production, please 
+#       consider a different approach of hosting. This setup can lead to 
+#       errors and loss of production data if handled incorrectly. Currently 
+
+# WARNING: Currently, container based setups are not officially supported by Humhub GmbH.
+#          Use at your own risk!
+version: '3.1'
+services:
+  nginx:
+    build:
+      context: .
+      target: humhub_nginx
+    ports:
+      - "8080:80"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost/ping"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+    volumes:
+      - "uploads:/var/www/localhost/htdocs/uploads"
+      - "assets:/var/www/localhost/htdocs/assets"
```

Comment:
add the following volumes:
"modules:/var/www/localhost/htdocs/protected/modules"
"themes:/var/www/localhost/htdocs/themes"